internal/metrics.go

```diff
@@ -33,12 +36,28 @@ func newMetrics() *metrics {
 		Name: "fleetlock_unlock_request_count",
 		Help: "Number of unlock requests",
 	})
+	totalRequests := prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "http_requests_total",
+		Help: "Number of requests",
+	}, []string{"path"})
+	responseStatus := prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "http_response_status",
+		Help: "Status of HTTP response",
+	}, []string{"path", "status"})
+	httpDuration := prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Name:    "http_response_time_millis",
+		Help:    "Duration of HTTP requests in milliseconds",
+		Buckets: prometheus.ExponentialBuckets(10e-9, 10, 10),
```

Comment:
Why not store it as seconds? 

As a point of comparison, there's 116,000+ metrics using seconds https://alerts.lyft.net/metrics?qs=%7B%22s%22%3A%5B%5D%7D&query=.*seconds.*&type=2
and only 179 using milliseconds https://alerts.lyft.net/metrics?qs=%7B%22s%22%3A%5B%5D%7D&query=.*milliseconds.*&type=2

Reply:
Most HTTP responses should be O(hundreds of milliseconds). I guess in this case since we can graph it in any unit we want it doesn't really matter.

I'll simplify it.

Reply:
agree that none will likely be in the seconds.